### PR TITLE
More CSS Changes to Master

### PR DESCRIPTION
**Pull request by zcollins0**
WIP.

Just putting all my CSS changes up so people can test them, and creating a PR so it's easily visible.

### CODE REVIEW

**Comment by kurtlewis** [comment]
FYI looks like you accidentally killed flash messages https://github.com/zacharysang/flock/blob/css/master/flock_server/templates/layout.html#L25-L28

**Comment by zcollins0** [comment]
Enough changes here that it might be worth merging now.

**Comment by zcollins0** [comment]
> FYI looks like you accidentally killed flash messages https://github.com/zacharysang/flock/blob/css/master/flock_server/templates/layout.html#L25-L28

I don't think I did...

File: master/flock_server/templates/host/submit.html [hunk 1]
```diff
@@ -3,31 +3,54 @@
 {% block title %}Submit a Project{% endblock %}
 {% block content %}
   <h1>Submit a Project</h1>
+  <p>Create a new project. After submitting your project information, it will be entered into a moderation queue for approval. You will be notified via email when the project is approved and automatically deployed.</p>
```
Comment (kurtlewis) [inline master/flock_server/templates/host/submit.html]: This isn't true. Need to remove the "you will be notified via email"

**Comment by zcollins0** [comment]
All changes addressed. Flashed messages aren't too pretty yet - I'll handle that sometime soon.

**Pull request merged**